src/app.js

```diff
@@ -24,9 +24,9 @@ export function app() {
   searchInput.addEventListener('input', event => {
     main.removeChild(pokemons);
 
-    const searchValue = event.target.value;
+    const searchValue = event.target.value.toUpperCase();
     const filteredPokemons = allPokemons.filter(pokemon => {
-      return pokemon.startsWith(searchValue);
+      return pokemon.toUpperCase().startsWith(searchValue);
```

Comment:
Why toUpperCase?
Shouldn't it be toLowerCase??

Reply:
It doesn't matter if upper- or lowercase. It just has to be the same for the input value and the pokemon 😀

Reply:
alrighty 👍